ocean-client/src/utils/store.ts

```diff
@@ -8,6 +8,16 @@ export class Store {
         this.settings = new StoredSettings()
     }
 
+    async updateToState(value: string): Promise<void> {
+        const state = {
+            Name: StoreKey.State,
+            Value: value,
+            Overwrite: true,
+            Type: 'String'
+        }
+        await this.ssm.putParameter(state).promise()
```

Comment:
currently the lambda only has rights to read the parameters, not set them. Do we really need a state?
And if yes: add the prefix to the state for multiple lambdas to still work.

Reply:
fair point about those permissions, maybe there is something in permissions to just allow writing into that one parameter.

yes did it for reading state, but missed it for updating